modules/guides/pages/optimize.adoc

```diff
@@ -0,0 +1,41 @@
+= Optimizing Performance
+:page-role: tiles -toc
+:page-pagination: next
+:description: This guide explains how to optimize the performance of N1QL queries using prepared statements, the index advisor, and the cost-based optimizer.
+:!sectids:
+
+// Pass through HTML styles for this page.
+
+ifdef::basebackend-html[]
```

Comment:
Ah, so this is the ugly hack for CSS styles?
It's good to know (or not, arguably) but happy to pair/review extracting this out to docs-ui as we did for the REST API stuff.

Reply:
Yes. I think this one has been tested enough that I should probably put it into docs-ui 😄